Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -166,6 +174,9 @@
     blacklist:
       tags:
       - NukeOpsUplink
+  - !type:BuyerSpeciesCondition
+    blacklist:
+    - Oni # goob
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)